src/components/UI/SimpleTable/SimpleTable.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
```

Comment:
можно сократить 7 строк в одну
```suggestion
import { Table, TableBody, TableCell, TableContainer, TableHead, TableRow, Paper } from '@material-ui/core';
```

Reply:
Исправлен